python/ml4ir/data/tfrecord_helper.py

```diff
@@ -0,0 +1,64 @@
+from tensorflow import train
+import tensorflow as tf
+from ml4ir.config.keys import TFRecordTypeKey
+
+
+def _bytes_feature(values):
```

Comment:
are these moved from elsewhere?

Reply:
Yep. I had to move it to a new module to avoid cyclic imports.

Reply:
Not new code.